dts/bindings/shi/nuvoton,npcx-shi.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022 Google Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX Serial Host Interface (SHI) node
+
+compatible: "nuvoton,npcx-shi"
+
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    description: mmio register space
+    required: true
+
+  clocks:
+    required: true
+    description: configurations of device source clock controller
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-1:
+    required: false
+
+  pinctrl-names:
+    required: true
+
+  shi-cs-wui:
+    type: phandle
+    required: true
+    description: |
+      Mapping table between Wake-Up Input (WUI) and SHI_CS signal.
+
+      For example the WUI mapping on NPCX7 would be
+        shi-cs-wui = <&wui_io53>;
+
+  buffer-rx-size:
+    type: int
+    required: true
+    description: |
+      Size of the internal buffer used for receiving data on the bus.
+
+  buffer-tx-size:
+    type: int
+    required: true
+    description: |
+      Size of the internal buffer used for transmitting data over the bus.
```

Comment:
Would it make sense to add these two to a common template here?

Reply:
Sure it does, done